### PR TITLE
Handle unsupported transaction

### PR DESCRIPTION
**Pull request by bludnic**
Handle unsupported transaction types (e.g. for [AIP-18: Decentralized file transfer](https://github.com/Adamant-im/AIPs/pull/54/files)) in old versions of the messenger.

image

We can introduce other types of transactions in the future, so it will break the app every time until we make a new release and update the client.

Added a fix in `normalizeMessage`.

### CODE REVIEW

**Comment by bludnic** [comment]
[Handle unsupported transactions](https://trello.com/c/ZXO4ZPca/543-handle-unsupported-transactions)

**Review by RealGoodProgrammer — APPROVED** [review]

File: src/locales/ru.json [hunk 1]
```diff
@@ -77,7 +77,8 @@
     "you_reacted": "Отреагировали",
     "partner_reacted": "Отреагировал",
     "you_removed_reaction": "Убрали реакцию",
-    "partner_removed_reaction": "Убрал реакцию"
+    "partner_removed_reaction": "Убрал реакцию",
+    "unsupported_transaction_type": "Данный тип транзакции не поддерживается. Пожалуйста, обновите приложение до последней версии."
```
Comment (RealGoodProgrammer) [inline src/locales/ru.json:81]:
```suggestion
    "unsupported_transaction_type": "Данный тип транзакции не поддерживается. Пожалуйста, обновите страницу, либо установите актуальную версию приложения."
```

File: src/locales/en.json [hunk 2]
```diff
@@ -77,7 +77,8 @@
     "you_reacted": "Reacted",
     "partner_reacted": "Reacted",
     "you_removed_reaction": "Removed reaction",
-    "partner_removed_reaction": "Removed reaction"
+    "partner_removed_reaction": "Removed reaction",
+    "unsupported_transaction_type": "This type of transaction is not supported. Please update the app to the latest version."
```
Comment (RealGoodProgrammer) [inline src/locales/en.json:81]:
```suggestion
    "unsupported_transaction_type": "This transaction type is not supported. Please refresh the page or install the latest version of the app."
```

**Comment by adamant-al** [comment]
It's better to show this message as _italic_ and in `{}` or `[]`.

**Comment by bludnic** [comment]
> It's better to show this message as _italic_ and in `{}` or `[]`.

Noted: https://trello.com/c/yxpIYa9Y/546-style-unsupported-transaction-type

**Comment by adamant-al** [comment]
@bludnic 
Check how we already show ~"Unable to decrypt transaction".

**Pull request merged**